### PR TITLE
Update terminus drush commands

### LINKED ISSUE #1232: Audit terminus drush commands
Update with changes made in terminus 0.10.0 release

https://github.com/pantheon-systems/cli/blob/master/CHANGELOG.md

### PR DESCRIPTION
**Pull request by nataliejeremy**
Closes #1232 
This PR adds quotes to commands passed through terminus drush, based on the Terminus 0.10.0 release.
@timani @rachelwhitton Please review--thanks!

### CODE REVIEW

**Comment by nataliejeremy** [comment]
@TeslaDethray Can you take a look as well? Thanks.

**Comment by bmackinney** [comment]
Needs work. Terminus drush and wp commands now cannot be interrupted by the `--site=<site>` and `--env=<env>` options. I think we should decide to delete those options from all example commands. Users can provide them interactively, learn to add them before running, or use a `.env` file.

File: source/docs/articles/drupal/8/configuration-management.md [hunk 1]
```diff
@@ -32,13 +32,13 @@ This keeps us from needing to use `--site=book-blog` with all of our terminus co
 
 ### Workflow Example
 
-1. `terminus drush cex -y --env=dev`
+1. `terminus drush 'cex -y --env=dev'`
```
Comment (TeslaDethray) [inline source/docs/articles/drupal/8/configuration-management.md]: This is not correct. the `--env=dev` piece of this belongs to Terminus, not Drush, so it should not be inside the quotes. It also needs a `--site=<site>` It should be
`terminus drush 'cex -y' --env=dev --site=<site>`

File: source/docs/articles/drupal/8/configuration-management.md [hunk 2]
```diff
@@ -32,13 +32,13 @@ This keeps us from needing to use `--site=book-blog` with all of our terminus co
...
 2. `terminus site code commit --env=dev --message="Export configuration to code"`
 3. `terminus site deploy --env=test --sync-content --cc --note="Deploy configuration to test"`
-4. `terminus drush cim -y --env=test`
+4. `terminus drush 'cim -y --env=test'`
```
Comment (TeslaDethray) [inline source/docs/articles/drupal/8/configuration-management.md]: ``terminus drush 'cim -y' --site=<site> --env=test`

File: source/docs/articles/drupal/8/configuration-management.md [hunk 3]
```diff
@@ -32,13 +32,13 @@ This keeps us from needing to use `--site=book-blog` with all of our terminus co
...
 5. `open http://test-mysite.pantheon.io`
 6. `terminus site deploy --env=live --cc --note="Deploy configuration to live"`
-7. `terminus drush cim -y --env=live`
+7. `terminus drush 'cim -y --env=live'`
```
Comment (TeslaDethray) [inline source/docs/articles/drupal/8/configuration-management.md]: `terminus drush 'cim -y' --site=<site> --env=live`

File: source/docs/articles/drupal/launch-check-drupal-performance-and-configuration-analysis.md [hunk 4]
```diff
@@ -45,7 +45,7 @@ The Dashboard integration is intended to provide developers with the most action
 
 You can get a list of all available site audit reports using [Terminus](/docs/articles/local/cli/):
 ```nohighlight
-terminus drush help --site=<site> --env=<env> --filter=site_audit
+terminus drush "help --site=<site> --env=<env> --filter=site_audit"
```
Comment (TeslaDethray) [inline source/docs/articles/drupal/launch-check-drupal-performance-and-configuration-analysis.md]: `terminus drush --site=<site> --env=<env>  "help --filter=site_audit"`

File: source/docs/articles/drupal/launch-check-drupal-performance-and-configuration-analysis.md [hunk 5]
```diff
@@ -53,7 +53,7 @@ Replace <code>&lt;site&gt;</code> with your site name, and <code>&lt;env&gt;</co
 
 You can also execute a full report in HTML format.
 ```bash
-terminus drush aa --site=<site> --env=<env> --skip=insights --html --bootstrap --detail --vendor=pantheon > report.html
+terminus drush "aa --site=<site> --env=<env> --skip=insights --html --bootstrap --detail --vendor=pantheon" > report.html
```
Comment (TeslaDethray) [inline source/docs/articles/drupal/launch-check-drupal-performance-and-configuration-analysis.md]: `terminus drush --site=<site> --env=<env> "aa --skip=insights --html --bootstrap --detail --vendor=pantheon" > report.html`

File: source/docs/articles/drupal/launch-check-drupal-performance-and-configuration-analysis.md [hunk 6]
```diff
@@ -77,7 +77,7 @@ Use the [Site Audit Issue Queue](https://drupal.org/project/issues/site_audit) t
 
 If your site's Launch Check is showing recent update information about Database or Redis usage, but older information for the Site Audit checks, and clicking "run the checks now" doesn't update the status, there may be an application error interrupting its complete operation. In order to debug what might be causing an error, you can run the [Terminus](/docs/articles/local/cli/) command to execute Site Audit directly on your Pantheon site:
 ```bash
-terminus drush --site=<site> --env=<env> -vd @pantheon.SITENAME.ENV aa --skip=insights --detail --vendor=pantheon --strict=0
+terminus drush --site=<site> --env=<env> -vd @pantheon.SITENAME.ENV "aa --skip=insights --detail --vendor=pantheon --strict=0"
```
Comment (TeslaDethray) [inline source/docs/articles/drupal/launch-check-drupal-performance-and-configuration-analysis.md]: `terminus drush --site=<site> --env=<env> "-vd @pantheon.SITENAME.ENV aa --skip=insights --detail --vendor=pantheon --strict=0"`

File: source/docs/articles/sites/code/email.md [hunk 7]
```diff
@@ -28,7 +28,7 @@ This is a common error with the SMTP module. It can be fixed in a few steps:
 1. Copy the file from .../files/mailsystem/filename.inc
 2. Place in a custom module's includes dir and .info file using files[] = includes/filename.inc
 3. Remove original file from {registry} table DELETE FROM registry WHERE name='[appropriate-name]' AND module='mailsystem';
-4. [`terminus drush --site=<site> --env=<env> cc all`](https://github.com/pantheon-systems/cli)
+4. [`terminus drush --site=<site> --env=<env> "cc all`](https://github.com/pantheon-systems/cli)"
```
Comment (TeslaDethray) [inline source/docs/articles/sites/code/email.md]: [`terminus drush --site=<site> --env=<env> "cc all"`](https://github.com/pantheon-systems/cli)

File: source/docs/articles/sites/timeouts.md [hunk 8]
```diff
@@ -98,7 +98,7 @@ Rules are for the good of the group, and timeouts are no exception. We've config
 
 #### Can I manually run Drupal cron for longer than the Pantheon executed Drupal cron?
 
-Yes, just use `terminus drush cron --site=<sitename> --env=<env>` using [Terminus](/docs/articles/local/cli/). With that said, most slow cron executions are due to PHP errors or a slow external service. Rather than throwing more resources at an inefficient process, determine why it's slow and fix the root cause.
+Yes, just use `terminus drush "cron --site=<sitename> --env=<env>"` using [Terminus](/docs/articles/local/cli/). With that said, most slow cron executions are due to PHP errors or a slow external service. Rather than throwing more resources at an inefficient process, determine why it's slow and fix the root cause.
```
Comment (TeslaDethray) [inline source/docs/articles/sites/timeouts.md]: `terminus drush "cron" --site=<site> --env=<env>`

File: source/docs/guides/terminus-drupal-site-management.md [hunk 9]
```diff
@@ -129,7 +129,7 @@ $ terminus sites show
 Now that the site is created, the next step is to run a Drush install command to get a fully functional Drupal set ready to go for development. Terminus will run most available Drush commands by simply adding the word "drush" to the command directly afterward, along with the site's Pantheon machine name.
 
 ```nohighlight
-$ terminus drush site-install --site=terminus-cli-create
+$ terminus drush "site-install --site=terminus-cli-create"
```
Comment (TeslaDethray) [inline source/docs/guides/terminus-drupal-site-management.md]: `terminus drush "site-install" --site=terminus-cli-create`

File: source/docs/guides/terminus-drupal-site-management.md [hunk 10]
```diff
@@ -172,13 +172,13 @@ $ terminus site environments --site=terminus-cli-create
 While the site's Dev environment is still in SFTP mode, we can use Drush to download and install some Drupal contrib modules, such as Views and Administration Menu.
 
 ```nohighlight
-$ terminus drush dl admin_menu --site=terminus-cli-create
+$ terminus drush "dl admin_menu --site=terminus-cli-create"
```
Comment (TeslaDethray) [inline source/docs/guides/terminus-drupal-site-management.md]: `terminus drush "dl admin_menu" --site=terminus-cli-create`

File: source/docs/guides/terminus-drupal-site-management.md [hunk 11]
```diff
@@ -172,13 +172,13 @@ $ terminus site environments --site=terminus-cli-create
...
 Running drush dl admin_menu  on terminus-cli-create-dev
 dev.a248f559-fab9-49cd-983c-f5@appserver.dev.a248f559-fab9-49cd-983c-f5c0d11a2464.drush.in's password:
 Project admin_menu (7.x-3.0-rc5) downloaded to                         [success]
 /srv/bindings/c183403f14224eac8471ec0000f9e653/code/sites/all/modules/admin_menu.
 Project admin_menu contains 3 modules: admin_devel, admin_menu_toolbar, admin_menu.
-tests-MacBook-Pro:~ erikmathy$ terminus drush en admin_menu,admin_menu_toolbar --site=terminus-cli-create
+tests-MacBook-Pro:~ erikmathy$ terminus drush "en admin_menu,admin_menu_toolbar --site=terminus-cli-create"
```
Comment (TeslaDethray) [inline source/docs/guides/terminus-drupal-site-management.md]: terminus drush "en admin_menu,admin_menu_toolbar" --site=terminus-cli-create

File: source/docs/guides/terminus-drupal-site-management.md [hunk 12]
```diff
@@ -211,7 +211,7 @@ Open the Pantheon Dashboard, and you'll see the new files are shown in the Git c
 To see what a commit message looks like, let's download Bootstrap and then commit it as well.
 
 ```nohighlight
-$ terminus drush dl bootstrap --site=terminus-cli-create --env=dev
+$ terminus drush "dl bootstrap --site=terminus-cli-create --env=dev"
```
Comment (TeslaDethray) [inline source/docs/guides/terminus-drupal-site-management.md]: terminus drush "dl bootstrap" --site=terminus-cli-create --env=dev

**Comment by nataliejeremy** [comment]
@rachelwhitton I incorporated the feedback from @TeslaDethray. Can you review?

**Pull request merged**